boot/bootdata/txtsetup.sif

```diff
@@ -186,6 +188,7 @@ pci = pci.sys
 isapnp = isapnp.sys
 
 [InputDevicesSupport.Load]
+usbxhci = usbxhci.sys
 usbehci = usbehci.sys
 usbohci = usbohci.sys
 usbuhci = usbuhci.sys
```

Comment:
Can/should this be ordered alphabetically as well? Placing the
usbxhci = usbxhci.sys
under
usbuhci = usbuhci.sys
????

Reply:
The reason why I did it that way was because it appeared to be in order that shows the progression of USB specification versions. I put usbxhci = usbxhci.sys above usbehci = usbehci.sys because it made sense i.e because EHCI is version 2.0 and XHCI is 3.0. 

I can change it if you want

Reply:
So you ordered from newest to oldest here. I see,
That would mean OHCI is younger than UHCI, which might be true. (I tend to always mix the two in my head.)

`I can change it if you want`
I don't care much. If you had your reasons and it was no oopsie, that is enough explanation for me. Leave it like you had in mind then.
Resolved.